### ISSUE TITLE
[UX] Consolidate the "File settings" and "File system" pages

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4121]
I went looking for the max file upload limit setting today, and by force of habit I initially navigated to _Configuration_ -> _Media_ -> _File system_, only to find that the setting was not there. I then of course navigated to _Structure_ -> _File types_ -> _File settings_, where I found the setting, but it felt that the settings in these two admin pages were overlapping in certain aspects.

Ideally, a person should not need to be a Drupal/Backdrop-savvy person to know where these things are, and it feels that perhaps they could live under the same "roof", or link from one page to the other(?).

I don't have a solution/proposal at this point; just wanted to quickly create this issue, so to start a discussion going on.

**Comment by Graham-72** [comment]
> Ideally, a person should not need to be a Drupal/Backdrop-savvy person to know where these things are.

I think this is an excellent principle that should now apply throughout Backdrop!

**Comment by laryn** [comment]
I agree it should be friendlier to move between these two pages -- whether that means merging them in some way or at least having some text at the top linking them together (so if you find one, you are presented with a direct link to the other).

A more intensive response could also be some sort of system that allows cross-referencing "related pages" more generally in a more automated way -- but that might be more than is required. For example - when editing a menu item, an additional selector for "Related pages" in a similar way that you can select the parent link; then adding a block to admin pages that lists related page(s).
